exchange/exchange_test.go

```diff
@@ -4539,9 +4540,9 @@ type capturingRequestBidder struct {
 	req *openrtb2.BidRequest
 }
 
-func (b *capturingRequestBidder) requestBid(ctx context.Context, bidderRequest BidderRequest, conversions currency.Conversions, reqInfo *adapters.ExtraRequestInfo, adsCertSigner adscert.Signer, bidRequestMetadata bidRequestOptions, alternateBidderCodes openrtb_ext.ExtAlternateBidderCodes, executor hookexecution.StageExecutor) (seatBid []*pbsOrtbSeatBid, errs []error) {
```

Comment:
Can you re-add the `executor hookexecution.StageExecutor` parameter so `capturingRequestBidder` satisfies the `AdaptedBidder` interface?